src/uct/tcp/tcp_iface.c

```diff
@@ -270,20 +270,44 @@ uct_tcp_iface_is_reachable_v2(const uct_iface_h tl_iface,
         return 0;
     }
 
+    if (ucs_netlink_route_exists(ndev_index,
+                                 (const struct sockaddr *)&remote_addr,
+                                 &has_default_gw)) {
+        return 1;
+    }
+
+    /* Going to search for a default GW route on this interface. If found, and
```

Comment:
Please move to lines 286-293 block, make more concise (w/o "going to..", only explain the reasoning)